ktlint/src/main/kotlin/com/pinterest/ktlint/internal/FileUtils.kt

```diff
@@ -28,9 +29,18 @@ internal fun List<String>.fileSequence(): Sequence<File> {
         .map(Path::toFile)
 }
 
+internal fun List<String>.toFilesURIList() = map {
```

Comment:
Maybe we should introduce some `typealias Rulesets` and have extension functions for that to avoid namespace pollution?

Reply:
Good idea, will do :+1: